packages/components/src/components/tabs/tabs.scss

```diff
@@ -1,22 +1,43 @@
 .dc-tabs {
-    &__list {
-        @include themify($themes) {
-            border-bottom: 1px solid themed('border_color');
+    position: relative;
+    @for $i from 1 through 4 {
```

Comment:
What is 1 through 4 representing?

Reply:
ooh damn! i forgot to use css var here

Reply:
@mahdipourziaei-binary i couldnt find a way to send the number in, i will use 5 here. i dont think we will have more